controllers/performanceprofile_controller.go

```diff
@@ -178,10 +178,10 @@ func validateUpdateEvent(e *event.UpdateEvent) bool {
 	return true
 }
 
-// +kubebuilder:rbac:groups="",resources=events,verbs=*
+// +kubebuilder:rbac:groups="",resources=events;nodes,verbs=*
```

Comment:
Please reduce the number of permissions, in general, `get,list,watch` should be enough, I think something like 
```
// +kubebuilder:rbac:groups="",resources=nodes,verbs=get,list,watch
```
should work